fgrpc/grpcrunner_test.go

```diff
@@ -95,6 +95,16 @@ func TestGRPCDestination(t *testing.T) {
 			"localhost:1234",
 			"localhost:1234",
 		},
+		{
+			"http hostname and port",
+			"http://localhost:1234",
+			"localhost:1234",
+		},
+		{
+			"https hostname and port",
+			"https://localhost:1234",
+			"localhost:1234",
```

Comment:
can we check grpc secure state ?

Reply:
that is not being set in the parseGRPCDestination function. It's tied in with the fgrpc.Dial function.

```
if tls || (strings.HasPrefix(serverAddr, "https://")) {
		opts = grpc.WithTransportCredentials(credentials.NewTLS(nil))
	}
```

not sure how to best test it.

Reply:
you can test it (at least manually, and probably also in Webtest.sh, with
fortio grpcping https://fortio.istio.io/
(it will need to set the port to 443 too ideally if not supported)

maybe we should just bail if it has http[s]:// prefix

Reply:
sorry, was unclear earlier. i had tested it manually and the https state led to secureTLS internal state. In any case, I added an additional grpcping in webtest.sh